lib/generators/rspec/integration/templates/api_request_spec.rb

```diff
@@ -11,59 +11,65 @@
 # It only uses APIs available in rails and/or rspec-rails.  There are a number
 # of tools you can use to make these specs even more expressive, but we're
 # sticking to rails and rspec-rails APIs to keep things simple and stable.
-#
-# Compared to earlier versions of this generator, there is very limited use of
-# stubs and message expectations in this spec.  Stubs are only used when there
-# is no simpler way to get a handle on the object needed for the example.
-# Message expectations are only used when there is no simpler way to specify
-# that an instance is receiving a specific message.
-#
-# Also compared to earlier versions of this generator, there are no longer any
-# expectations of assigns and templates rendered. These features have been
-# removed from Rails core in Rails 5, but can be added back in via the
-# `rails-controller-testing` gem.
+
```

Comment:
You can't just remove existing functionality, if you wish to add more functionality thats fine but these files must remain as is, further comments assume you've done this and apply to a new file containing functionality.